File: MiniPrograms/CPP/EvenOrOdd.cpp
#include <iostream>
using namespace std;

int main()
{
    int num;
    cout << "Type an integer: " << endl;
    cin >> num;
    
    // Modulus (%) gets remainder
    if (num % 2 == 0)
    {
        cout << num << " is an even number." << endl;
    }
    else
    {
        cout << num << " is an odd number." << endl;
    }

    return 0;
}

File: Go/EvenOrOdd.go
package main

import "fmt"

func main() {
	var num int
	fmt.Print("Enter an integer: ")
	fmt.Scanln(&num)

	// Use modulus
	if num%2 == 0 {
		fmt.Println("The number you typed is even.")
	} else {
		fmt.Println("The number you typed is odd.")
	}
}
